experimental/Schemes/IdealSheaves.jl

```diff
@@ -873,7 +876,12 @@ function radical_membership(x::RingElem, I::MPolyQuoLocalizedIdeal)
 end
 
 # Proof of concept method
-function _maximal_associated_points(I::AbsIdealSheaf; covering=default_covering(scheme(I)), use_decomposition_info::Bool=true)
+function _maximal_associated_points(
+    I::AbsIdealSheaf; 
+    covering=default_covering(scheme(I)), 
+    use_decomposition_info::Bool=true,
+    algorithm::Symbol=:GTZ
+  )
```

Comment:
Quoting the singular manual:

https://www.singular.uni-kl.de/Manual/latest/sing_1426.htm#SEC1507
"In small characteristic and for algebraic extensions, the procedures via Gianni, Trager, Zacharias may not terminate. "

Reply:
Better link:
https://www.singular.uni-kl.de/Manual/latest/sing_1511.htm#SEC1592

Reply:
The algebraic extensions only refer to those implemented as extended basefield in Singular. If you encode the algebraic extension by hand, the problem does not occur (hence the encoding by hand).  This is the reason why these particular calls to Singular should not use the algebraic extensions in Singular, but do the encoding themselves (this was subject of discussion in another PR). 
The restriction on the characteristic results from the sufficiently general linear coordinate change, which is used (the open set needs to be non-empty, which might be a problem in very small characteristics). This should be stated as a warning somewhere in the docstring, but allowing characteristics like 101 or 32003 is vital for many practical applications where coefficients are known to explode in characteristic zero.